pkgs/by-name/pd/pdfid/package.nix

```diff
@@ -25,7 +25,8 @@ python3Packages.buildPythonApplication rec {
     runHook preInstall
     mkdir -p $out/{bin,share/pdfid}
     cp -a * $out/share/pdfid/
-    makeBinaryWrapper ${lib.getExe python3} $out/bin/${meta.mainProgram} \
+    makeWrapper ${lib.getExe python3} $out/bin/${meta.mainProgram} \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
```

Comment:
Is this really required ?

Reply:
To have a working binary in `$out/bin`, yes.